pkgs/applications/networking/feedreaders/newsboat/default.nix

```diff
@@ -59,12 +59,12 @@ rustPlatform.buildRustPackage rec {
     updateScript = nix-update-script { };
   };
 
-  meta = with lib; {
+  meta = {
     homepage    = "https://newsboat.org/";
     changelog   = "https://github.com/newsboat/newsboat/blob/${src.rev}/CHANGELOG.md";
     description = "A fork of Newsbeuter, an RSS/Atom feed reader for the text console";
-    maintainers = with maintainers; [ dotlambda nicknovitski ];
-    license     = licenses.mit;
-    platforms   = platforms.unix;
+    maintainers = with lib.maintainers; [ dotlambda nicknovitski ];
+    license     = lib.licenses.mit;
+    platforms   = lib.platforms.unix;
   };
```

Comment:
why are you removing `with lib;` here? In this scope it's fine we just don't want it in the entire package scope so at start of the file

Reply:
Thanks for the feedback, @Janik-Haag, much appreciated. I see your point, yet I've seeing `with lib` being removed from the meta section in other packages, e.g. gst123, openxcom. Might be an overzealous misunderstanding from my end.
Are there any explicit nixpkgs guidelines regarding `with lib`, etc. that you could point me to?

Reply:
> Happy to be pointed to resources for best practices in regards to "improving" packages to comply with them in future PRs.

From what I know we don't really have formal guide lines for the most part except for [nixpkgs-manual](https://nixos.org/manual/nixpkgs/unstable/) and https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md which I guess you bot already know. The other things I know is basically some more experienced contributor told me at some point or I asked in the nixpkgs dev matrix channel and we came to some conclusion after discussing the topic.

Other then that I think it's fine to do it like this here but I would also be fine with having it for the entire meta scope.